### PR TITLE
Fix Edge Functions - exclude ELEVENTYEDGE from minification

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Excludes the compiled `<!-- ELEVENTYEDGE -->` comments form html minification, which fixes Edge Functions hosted on Netlify.

Also fixes the gitignore to use `/generated/` rather than `/generated` which is generated at build time by Eleventy Edge & Netlify.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -500,6 +500,9 @@ module.exports = function(eleventyConfig) {
                     conservativeCollapse: true,
                     preserveLineBreaks: true,
                     removeComments: true,
+                    ignoreCustomComments: [
+                        /^\s+ELEVENTYEDGE/
```
Comment (ZJvandeWeg) [inline .eleventy.js:504]: I think more simple would be:

```suggestion
                        /\<!-- ELEVENTYEDGE --\>/
```

Comment (joepavitt) [reply .eleventy.js:504]: If that was the full comment, yes, but an example full comment is: 

`<!--ELEVENTYEDGE_edge "njk" %}EleventyEdgeNunjucksPrecompile:349b99eaaab03aab275b22b91bad65b5a21877cc28a5a6db3f487f3367129c80ELEVENTYEDGE_edge-->`

Comment (joepavitt) [reply .eleventy.js:504]: This regex (as things stand doesn't actually work - was meant to open as Draft)

**Comment by ZJvandeWeg** [comment]
As this is the solution that was proposed by Pez in Slack, I'm going ahead and merge this without his explicit review.

**Comment by Pezmc** [comment]
@ZJvandeWeg "This regex (as things stand doesn't actually work - was meant to open as Draft)"

**Pull request merged**